lib/membrane_mp4/muxer/cmaf.ex

```diff
@@ -7,15 +7,21 @@ defmodule Membrane.MP4.Muxer.CMAF do
   Given that all input streams need to have a keyframe at the beginning of each CMAF Segment, it is recommended
   that all input streams are renditions of the same content.
 
+  The muxer also supports creation of partial segments that do not begin with a key frame
+  when the `partial_segment_duration_range` is specified (for best results when multiplied by an integer
+  it should sum to segment duration). The muxer tries to create many partial segments to match the
```

Comment:
Does it make sense if it doesn't sum to segment duration? Maybe sth like amount_of_partial_segments would be better?

Reply:
To be honest, the `for best results` probably no longer applies as the line was written when I hadn't had the range instead of raw durations implemented.  The ranges don't have to be a multiple of each other as the durations of partial segments can vary greatly depending on when the keyframe arrives.